gds_py/Dockerfile

```diff
@@ -8,8 +8,11 @@ ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
 
 #--- Python ---#
 
-RUN conda update --yes conda 
-RUN conda install -c conda-forge -c defaults --quiet --yes \
+RUN conda update conda --yes \
+    && conda config --add channels conda-forge \
+    && conda config --set channel_priority strict \
+    && conda install --yes --quiet \
+     'python=3.7'
```

Comment:
I prefer to specify the python version to ensure I know which one will be installed. I can remove this if you prefer not to specify.